cmd/edgectl/aes_install.go

```diff
@@ -612,7 +629,7 @@ func (i *Installer) Perform(kcontext string) error {
 		return err
 	}
 	i.Report("cluster_accessible")
-	i.show.Println("Your AES installation's address is", i.address)
+	i.show.Println("Your AES installation's address is", chalk.Bold.TextStyle(i.address))
```

Comment:
Can we drop `chalk` and use `gookit/color`  for everything? These two dependencies overlap in functionality, and I prefer `gookit/color` because it supports Windows.

Reply:
I use both because chalk allows different functionality (e.g. changing a word within a sentence to bold without terminating the style change at that point) and they interoperate well.  I will spend some more time to see if I can rewrite the code to properly work with just one or the other.

Reply:
Windows support is a requirement. If `chalk` doesn't _break_ Windows (and just leaves things unstyled in that case), then I suppose that's okay. Also, it looks like `chalk` might be unmaintained.

Reply:
latest push resolves this--`chalk` is no longer a requirement.